general_rules/ExternalComputing.tex

```diff
@@ -26,6 +26,34 @@ \section{External computing}\label{rule:robot_external_computing}
 
 \textbf{Remark:} Teams are allowed to use their own software in the external computing devices (not only cloud services). This software must be publicly available to other teams for scientific purposes (evaluation, test, and benchmarking), as well as for TC for inspection. Although open-sourcing the software is not mandatory, this practice is advised and encouraged by the league.
 
+\subsection{External Computing Devices on site for SSPL (Hardware)}
```

Comment:
Let's generalize to all leagues. This subsection is then renamed to:
*On-site External Computing Devices*

Reply:
Shouldn't the availability of software to other teams be after the competition? Otherwise, a team could potentially copy another team's software.

Reply:
No.

To breach the gap between Cloud Computing and External Computing, software resources (not source code) must be made available to all. If you have an outstanding ASR I should be able to try it also as is in your servers. We want to promote sharing and knowledge exchange in the league.

Any comments @LoyVanBeek? @balkce? @swachsmu? @dirkholz?